src/aardvark/aardvark_scene_graph.cpp

```diff
@@ -80,6 +80,7 @@ namespace aardvark
 	void from_json( const nlohmann::json& j, GadgetParams_t& params )
 	{
 		j.at( "uri" ).get_to( params.uri );
+		if(params.uri.back() == '/') params.uri.pop_back();
```

Comment:
Maybe add the same function on the C++ side? There are likely other places that will need this. (Or already have it.)

Reply:
Found the origin of the bad data. Looks like this function doesn't resolve to the correct format (at least in firefox / chrome) https://github.com/fuag15/aardvark/blob/e18092c552437bc817a2cc0794de9b34ff20ec1b/packages/aardvark-react/src/aardvark_gadget.tsx#L70-L82 What's your preference with fixing this function vs adding some data validation in the c++ side

Reply:
Moved it to a helper func and changed the source of the data to send the uri without a / at the end debated putting an initializer on that struct but didn't see that pattern around so opted for a free func in the aardvark namespace